src/lib/components/Content.svelte

```diff
@@ -25,7 +31,11 @@
     <slot name="toolbar-end" slot="toolbar-end" />
   </Header>
 
-  <div class="scrollable-content" class:open={$layoutMenuOpen}>
+  <div
+    class="scrollable-content"
+    class:open={$layoutMenuOpen}
+    on:scroll={handleScroll}
```

Comment:
I know this PR is a draft, but as early feedback, why are you hooking into scroll? It's a bit suboptimal in terms of performance. Assuming you want to hide something at the top on scroll, can't you use an Intersection Observer?

Reply:
Thanks for your early feedback. The initial plan was to hide on scroll down and display it back on scroll up. Not sure it's doable with the Intersection Observer.

Reply:
Ah to hide when it should be still visible on screen? Indeed it's a bit more tricky. Nvm my comment then. Thought you were building something like this.

<img width="1536" alt="Capture d’écran 2025-02-17 à 17 40 02" src="https://github.com/user-attachments/assets/42b3e311-93e5-4745-aea4-47b6deebfccf" />
<img width="1536" alt="Capture d’écran 2025-02-17 à 17 39 47" src="https://github.com/user-attachments/assets/8240d5b5-fc1d-4155-9f81-a4aa9a9a320a" />

Reply:
Actually something like this https://juno.build/docs/guides/nextjs

Reply:
Gotcha, that's Docusaurus.

But maybe like this too https://recyclingcredit.org/ ?

That's dapp I built with Svelte for the foundation - i.e. we have the source code, if interesting.